taichi/backends/cpu/jit_cpu.cpp

```diff
@@ -185,14 +187,19 @@ class JITSessionCPU : public JITSession {
   }
 
  private:
-  static void global_optimize_module_cpu(llvm::Module *module);
+  static void global_optimize_module_cpu(llvm::Module *module,
```

Comment:
We don't have to keep it as a static member function. Then we don't need these additional arguments.

Reply:
Reset this function as member function?

Reply:
Yes.